base/common/src/test/java/com/netscape/certsrv/cert/CertRevokeRequestTest.java

```diff
@@ -15,7 +14,7 @@
 
     @Before
     public void setUpBefore() {
-        before.setReason(RevocationReason.CERTIFICATE_HOLD);
+    //  before.setReason(RevocationReason.CERTIFICATE_HOLD);
```

Comment:
@edewata `RevocationReason` doesn't serialise to JSON because it has no JSON mappers, shall I make a PR in `JSS` to swap the XML mapper dependencies out for the Jackson ones? `RevocationReason` is the only class in `JSS` that has this XML serialisation code, so I think it's a small change (and for the better).

Reply:
I think so, and we probably could drop `RevocationReasonAdapter` and JAXB as well. However, JSS CI probably would fail since that class is still used in `CertRevokeRequest`, so the changes might need to be done in these steps:
* drop XML mapper from `CertRevokeRequest`
* drop JAXB from JSS and add JSON mapper for `RevocationReason`
* fix `CertRevokeRequestTest`

I think there's an existing JSS CI failure due to PKCS11 constants, but I think we can ignore it for now, and fix it in a separate PR.

Reply:
OK, so part 1 is #3620. Once that PR is merged OI can finish https://github.com/dogtagpki/jss/pull/785 and merge that, then fix the test. I will merge this in now, thanks!